packages/dds/tree/src/simple-tree/api/schemaCreationUtilities.ts

```diff
@@ -9,27 +9,33 @@ import { fail } from "../../util/index.js";
 
 import type { SchemaFactory, ScopedSchemaName } from "./schemaFactory.js";
 import type { NodeFromSchema } from "../schemaTypes.js";
-import type { NodeKind, TreeNodeSchemaClass, TreeNode } from "../core/index.js";
 import type {
-	InsertableObjectFromSchemaRecord,
-	ObjectFromSchemaRecord,
-} from "../objectNode.js";
-
-/**
- * Empty Object for use in type computations that should contribute no fields when `&`ed with another type.
- * @internal
+	InternalTreeNode,
+	NodeKind,
+	TreeNode,
+	TreeNodeSchemaClass,
+} from "../core/index.js";
+
+/*
+ * This file does two things:
+ *
+ * 1. Provides tools for making schema for cases like enums.
+ *
+ * 2. Demonstrates the kinds of schema utilities apps can write.
+ * Nothing in here needs access to package internal APIs.
+ * TODO:
```

Comment:
Nit (to disassociate the TODOs with list item 2
```suggestion
 * 
 * TODO:
```

Reply:
Or should these even go in a `@privateRemarks`?

Reply:
This is a file level comment, not a doc comment, so its not showing up in any API docs and tags don't seem needed. Format fix is good though.

Reply:
Actually, this got implemented a while back, so I'll just remove it.

Reply:
done.